lime/_backend/html5/HTML5Window.hx

```diff
@@ -480,6 +483,8 @@ class HTML5Window {
 				
 				case "mouseup":
 					
+					event.stopPropagation ();
```

Comment:
I see what you mean, so we need something like 
```
if (event.target == element) {
     event.stopPropagation ();
}
```
right ?

Reply:
Yeah, I think we need something like that, so it prevents a double mouse up event (I assume is why it is here?) but doesn't conflict with unrelated events.

Also, I forget if we handle the mouse up behavior in the Lime level, or OpenFL level, but I believe we dispatch a mouse up event for native _only_ if you were holding the mouse down when you left the window. If you leave the window and keep clicking outside the window, we don't fire events. This might require some testing?

Reply:
As far as I tested it shouldn't happen. But I can do something else to make sure that this never happen. Something like this:
```case "mousedown":
					
					if (event.target == element) {
						
						// Release outside browser window
						Browser.window.addEventListener("mouseup", handleMouseEvent);
						
					}
....
....
case "mouseup":
				Browser.window.removeEventListener("mouseup", handleMouseEvent);

					if (event.target == element) {
						
						event.stopPropagation ();
						
					}
```
This way we make sure that the listener is added only when element is mousedown-ed. Though I don't like a bit the adding and removing listeners every time when mousedown/up.
What do you think ?